admindb/sqlite/new.go

```diff
@@ -1,5 +1,19 @@
 // SPDX-License-Identifier: MIT
 
+// Package sqlite implements the SQLite backend of the admindb interfaces.
+//
+// It uses sql-migrate (github.com/rubenv/sql-migrate) for it's schema definition and maintainace.
+// For query construction/ORM it uses SQLBoiler (https://github.com/volatiletech/sqlboiler).
+//
+// The process of updating the schema and ORM can be summarized as follows:
```

Comment:
I think I never spelled out how updating the sqlite impl was supposed to go. Hope this makes it clearer.